packages/airnode-node/src/handlers/start-coordinator.test.ts

```diff
@@ -43,6 +44,7 @@ describe('startCoordinator', () => {
         },
       })),
     };
+
     jest.spyOn(fs, 'readFileSync').mockReturnValue(JSON.stringify(config));
```

Comment:
It's not entirely clear to me how the original idea of changing `mockReturnValue` to `mockReturnValueOnce`  would help with the `console` issue?

It looks like we cannot change it here or in `actions.test.ts` as  the tests don't pass in Github (although it works locally).

Reply:
The problem is that jest uses it's own logger and replaces the standard nodeJS console.log. The problem is that jest internally initializes this logger by a config file. To load this file it uses `readFileSync` which we mock and the return value is not what the logger expects and throws.

Reply:
Ah, I see! Although using mockReturnValueOnce seems to work locally, it doesn't pass on Github, weird.

Reply:
That's strange. Maybe some cache thing or something?